scripts/installDependencies.sh

```diff
@@ -1,30 +1,30 @@
+#!/bin/bash
+
 # Install nodejs dependencies
 echo "Current DIR: $PWD"
 # Script to install Node.js using NVM on Ubuntu
 
 # Read node version from .nvmrc file
-node_version=$(cat .nvmrc | tr -d "\n")
+node_version=$(cat $PWD/../.nvmrc | tr -d "\n")
```

Comment:
i think the current dir is not app dir, you need to `cd` into dir like https://github.com/LiskHQ/across-relayer/blob/9b5b68944336a1dc15b71bb137a12343e00bb473/scripts/installApplication.sh#L4-L6